### DISCUSSION TITLE
Cannot launch "flet" from command-line on macOS

### DISCUSSION THREAD

**Discussion by maziarzamani** [flet-dev/flet#397] [Q&A] [answered]
Hi everyone.

I have installed flet with pip, however I cannot type "flet" in terminal. 

Furthermore I was unable to find "flet" in `/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/site-packages/flet`.

Cheers, M.

**Comment by FeodorFitsner** [comment, marked as answer]
What about other CLI tools built with Python? Are you able to install them with pip and use via command line?

**Reply by maziarzamani** [reply]
> What about other CLI tools built with Python? Are you able to install them with pip and use via command line?

Hi @FeodorFitsner. You have a pretty good point here. I just tried installing Django, and I have the exact same problem.

**Reply by FeodorFitsner** [reply]
Have you solved that yet?

**Reply by maziarzamani** [reply]
Yes! I realized that I forgot to update my shell profile after I installed a newer version of Python. It works now. Thanks..
